forum/users/models.py

```diff
@@ -1,3 +1,50 @@
+from django.contrib.auth.models import AbstractBaseUser, BaseUserManager
 from django.db import models
+from django.utils import timezone
 
-# Create your models here.
+
+class CustomUserManager(BaseUserManager):
+    def create_user(self, email, password, **extra_fields):
+        if not email:
+            raise ValueError("The Email must be set")
+        email = self.normalize_email(email)
+        user = self.model(email=email, **extra_fields)
+        user.set_password(password)
+        user.save()
+        return user
+
+    def create_superuser(self, email, password, **extra_fields):
+        extra_fields.setdefault("is_staff", True)
```

Comment:
Code lines where mentioned `is_superuser` should be deleted, cause we get errors while trying to create a user.
Because `is_superuser` is not defined in the model